client/src/wh/logged_in/profile/components.cljc

```diff
@@ -747,12 +747,12 @@
 
 ;;height do nothing here, not figuring out how to make this section higher
 (defn section-cv [url]
-  [section
+  [section (util/smc styles/cv) 
    [sec-title "CV"]
-   [:iframe {:src url
-             :title "CV"
-             :height "400px"
-             :width "100%"}]])
+    [:div {:class styles/cv}
+     [:iframe {:src url
+              :title "CV"
+              :class styles/cv-iframe}]]])
```

Comment:
Do you need `styles/cv` on both of the `div` and the `section`? (I suspect you don't)